README.md

```diff
@@ -29,8 +41,8 @@ cd s2spy
 python3 -m pip install .
 ```
 
-## Configure the package for development and testing
-The testing framework used here is [pytest](https://pytest.org). Before running the test, the package need to be installed and configured as via the command:
+### Configure the package for development and testing
+The testing framework used here is [pytest](https://pytest.org). Before running the test, the package need to be installed and configured via the command:
```

Comment:
```suggestion
The testing framework used here is [pytest](https://pytest.org). Before running the test, we get a local copy of the source code via the command:

git clone https://github.com/AI4S2S/s2spy.git
cd s2spy

Then, the package needs to be installed:
```

Reply:
Please add code-block syntax ` ```py ...```  `around the codes. I couldn't add them while suggesting them here.

Reply:
Corrected!